drm-ffi/Cargo.toml

```diff
@@ -5,7 +5,7 @@ repository = "https://github.com/Smithay/drm-rs"
 version = "0.8.0"
 license = "MIT"
 authors = ["Tyler Slabinski <tslabinski@slabity.net>"]
-rust-version = "1.65"
+rust-version = "1.66"
```

Comment:
I've decided to bump rust-version everywhere, since CI does not distinguish between per-crate Rust versions. It shouldn't make a big difference and this way one can be certain that the MSRV is actually correct.